.github/workflows/go-short.yml

```
@@ -0,0 +1,21 @@
+name: Go Short Test
+on: [push]
+jobs:
+  quicktest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22.x"
+      - name: Run Revive Linter
+        uses: morphy2k/revive-action@v2
+        with:
+          config: revive.toml
```

@eeeady I found that this action has stopped working properly for some reason.  I don't know how/why they broke it, but there really is no reason to depend on it if they aren't going to be responsible authors and ensure releases work.  As such, I've changed this in my current project to use this instead:
```suggestion
- name: Install RunRevive Linter
        run: go install -v github.com/mgechev/revive@latest
      - name: Run Revive Linter
        run: revive -formatter friendly -config revive.toml ./...
```